kythe/cxx/indexer/textproto/testdata/generated_imports.pbtxt

```diff
@@ -0,0 +1,8 @@
+# proto-message: example2.OuterMessage
+# proto-file: kythe/cxx/indexer/textproto/testdata/imports_generated.proto
+
+#- @inner_msg ref OuterMessageInnerMsg
+inner_msg {
+    #- @field1 ref Field1
```

Comment:
Are these IDs referenced in other parts of the text?

Reply:
I don't know; I literally copied the extant imports test and changed the paths.

Reply:
(Checking: yes, they are mentioned by the copied example.proto -> generated.proto file)